.github/workflows/enforce-changelog-labels.yml

```diff
@@ -0,0 +1,44 @@
+name: Enforce changelog labels
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled]
+    branches: [master]
+
+jobs:
+  enforce-changelog-label:
+    runs-on: ubuntu-latest
+    env:
+      APPROVAL_MESSAGE: "The PR has a valid changelog label."
+      REJECTION_MESSAGE: "The PR is missing a valid changelog label."
+      # When updating the labels here, also update the `configure-sections` of the `.github_changelog_generator` file
+      ONE_OF_LABELS: "api|enhancement|breaking|bug|chore|documentation"
+    steps:
+      - name: Check the PR for a changelog label
+        id: check-changelog-label
+        continue-on-error: true
+        run: |
+          set -o xtrace
+          # Using the issues API instead of pulls because it can return only the labels
+          curl "https://api.github.com/repos/omgnetwork/elixir-omg/issues/${{ github.event.pull_request.number }}/labels" \
```

Comment:
minor: consider to use env var `GITHUB_REPOSITORY` to replace the `omgnetwork/elixir-omg` part to make the GH action script more portable.

Reply:
https://github.com/omgnetwork/elixir-omg/pull/1708/commits/32fcd381b756529f5a52629b5f056482b30d7a9f